packages/client/yggdrasil/yggdrasil.spec

```diff
@@ -2,9 +2,13 @@
 
 Name:    yggdrasil
 Version: 0.2.3
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Message dispatch agent for cloud-connected systems
+%if 0%{?suse_version}
```

Comment:
For what it's worth, Red Hat is also moving to SPDX tags. We haven't started the bulk migration, but see https://communityblog.fedoraproject.org/important-changes-to-software-license-information-in-fedora-packages-spdx-and-more/ for more. https://docs.fedoraproject.org/en-US/legal/update-existing-packages/ is the official Fedora way, but RHEL is planning the same. I just don't know for sure where that migration currently is. IIRC we've been asked internally to start changing.

It would be my preference to keep things simple. Can we change all the license tags for the client packages at once?